EXILED/Exiled.API/Features/Player.cs

```diff
@@ -506,7 +506,12 @@ public Player Cuffer
         public Vector3 Position
         {
             get => Transform.position;
-            set => ReferenceHub.TryOverridePosition(value, Vector3.zero);
+            set
+            {
+                ReferenceHub.TryOverridePosition(value, Vector3.zero);
+                if (Role is FpcRole fpcRole)
+                    fpcRole.RelativePosition = new(value);
```

Comment:
Will be better to exec that code only for npc, bcz i dont like the idea of forcing that for all players

Reply:
> Will be better to exec that code only for npc, bcz i dont like the idea of forcing that for all players

oh wait i was thinking making it NPC class oups